test/test_files/tinysnb/uuid/simpleTest.test

```diff
@@ -0,0 +1,17 @@
+-GROUP UUIDSimpleTest
+-DATASET CSV empty
+
+--
+
+-CASE UUIDCreateInsertMatch
```

Comment:
You should reuse the tinysnb dataset while doing such tests.
You can add tests under the existing update test file.